model/group.go

```diff
@@ -41,6 +41,10 @@ type Group struct {
 	DeleteAt     int64       `json:"delete_at"`
 	HasSyncables bool        `db:"-" json:"has_syncables"`
 	MemberCount  *int        `db:"-" json:"member_count,omitempty"`
+
+	// SchemeAdmin is significant only if the group is retrieved from the perspective of an associated syncable.
+	// Ex: "get groups by channel" or "get groups by team".
+	SchemeAdmin *bool `db:"-" json:"scheme_admin"`
```

Comment:
The alternative to having this field on `Group` is to create a new struct just for retrieving groups by channel or groups by team and then embed `Group` in that new struct, along with the `SchemeAdmin` field.